dockerfiles/jupyter/Dockerfile

```diff
@@ -65,7 +65,7 @@ RUN cd /tmp/mlrun && python -m pip install ".[complete-api]"
 
 # This will usually cause a cache miss - so keep it in the last layers
 ARG MLRUN_CACHE_DATE=initial
-RUN git clone --branch 1.5.x https://github.com/mlrun/demos.git $HOME/demos && \
+RUN git clone --branch 1.6.x https://github.com/mlrun/demos.git $HOME/demos && \
```

Comment:
we need to add this to the branch out procedure, easy to miss

Reply:
will add